examples/AccessControl/AccessControl.ino

```diff
@@ -429,19 +429,34 @@ void deleteID( byte a[] ) {
 }
 
 ///////////////////////////////////////// Check Bytes   ///////////////////////////////////
+/**
+This function cause some Card with 0-index value equals to 0 not found..then the programm just add fullstack of the same card.
```

Comment:
well with your commit problem is solved, so the comment is not useful
comments above functions should describe what the function do.